config/routes.rb

```diff
@@ -2,9 +2,9 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
-  root to: 'movies#index'
-  resources 'aws_texts' , only:[:index, :show]
-  resources 'questions'
-  resources 'lines', only: [:index]
-  resources 'movies', only:[:index]
+  root to: "movies#index"
+  resources "aws_texts", only: [:index, :show]
+  resources "questions", only: [:index, :create]
```

Comment:
詳細ページへのルーティングも必要です。これがありませんとエラーが出ます。

```suggestion
  resources "questions", only: [:index, :create, :show]
```